ex_2.14.c

```diff
@@ -0,0 +1,17 @@
+#include <stdio.h>
+#include <math.h>
+
+int main() {
+    float a, b, c;
+    printf("a = \n");
+    scanf("%f", &a);
+    printf("b = \n");
+    scanf("%f", &b);
+    printf("c = \n");
+    scanf("%f", &c);
+    float dis = (b * b) - (4 * a * c);

```

Comment:
dis<0?

Reply:
but there was condition in the task, that both root exist